tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.AssemblyReferencesHaveNotChanged.netcoreapp3.1.verified.txt

```diff
@@ -5,6 +5,7 @@ Microsoft.AspNetCore.Mvc.Abstractions, Version=2.0.0.0, Culture=neutral, PublicK
 Microsoft.AspNetCore.Routing, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
 Microsoft.AspNetCore.Routing.Abstractions, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
 Microsoft.Extensions.Primitives, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
+System.Buffers, Version=4.0.2.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
```

Comment:
This concerns me a bit, we don't want to add new assembly references ideally, can we avoid them? 😟

Reply:
It looks like this is introduced in the `MethodScopeMembers` type. We're considering removing the `ArrayPool` optimization in this PR so we can remove this new assembly reference, and we can vendor in a better array pooling mechanism in a later PR. WDYT?

Reply:
Great point! Removed this dependency in 763d90d21e1196804ea3a930909ac0678b17206b.